bims/templates/map.html

```diff
@@ -104,24 +105,22 @@
                     COLLECTOR <i class="fa fa-angle-up pull-right filter-icon-arrow"></i>
                 </div>
             </div>
-            <div class="row">
+            <div class="row" style="display: none">
                 <div class="col-lg-12 filter-content">
-                    <input type="checkbox" name="collector1" value="collector1"> Collector 1<br>
-                    <input type="checkbox" name="collector2" value="collector2"> Collector 2<br>
-                    <input type="checkbox" name="collector3" value="collector3"> Collector 3<br>
-                    <input type="checkbox" name="collector4" value="collector4"> Collector 4<br>
-                    <input type="checkbox" name="collector5" value="collector5"> Collector 5<br>
+                    {% for collector in collectors %}
```

Comment:
this process will taking long if list is big. Please see @dimasciput and my comment to avoid it.